src/nginxconfig/generators/conf/nginx.conf.js

```diff
@@ -196,6 +196,15 @@ export default (domains, global) => {
             'default': 'upgrade',
             '""': 'close',
         }]);
+        config.http.push(['map $remote_addr $proxy_forwarded_elem', {
```

Comment:
```suggestion
        // See https://www.nginx.com/resources/wiki/start/topics/examples/forwarded/
        config.http.push(['# Forwarded header coexisting with X-Forwarded-*', '']);
        config.http.push(['map $remote_addr $proxy_forwarded_elem', {
```
Add a note in the config itself about what the maps are for, and a comment in the JS as a ref for future contributors :)